cpu/ppc64_cpu_test.py

```diff
@@ -147,6 +152,14 @@ def dscr(self):
         command2 = "cat /sys/devices/system/cpu/dscr_default"
         self.equality_check("DSCR", command1, command2)
 
+    def smt_loop(self):
+        """
+        Tests smt on/off in a loop
+        """
+        for _ in range(1, 100):
+            process.run("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
+                        shell=True)
```

Comment:
SMT on/off command probably will not fail as far as the tool is concerned. If there are any failures it probably will be kernel failures. That was one of the reason I added the dmesg log collection after the loop test so that any kernel side failures/trace messages can be captured.